app/controllers/admin_controller.rb

```diff
@@ -33,6 +33,60 @@ def create_dispatcher
     end
   end
 
+  def edit_admin
+    admin = Admin.find(params[:id])
+    if admin.update_attributes(params_admin)
+      render json: { 'The admin is successfully updated!' => { name: admin.name, email: admin.email } }
```

Comment:
@abapolov it's better to respond with the full information about a user.
You can use https://github.com/rakvium/taxi-api/commit/055637ac5dc9fcd94752516e07c95d932afab7a4#diff-8394f3a652a395b55691e9fa1176eb28R21 as an example.